Globes.js

```diff
@@ -0,0 +1,248 @@
+	"label": "Globes",
+	"creator": "Eran Rosenthal",
+	"target": "^https?://www\\.globes\\.co\\.il/news/article.aspx",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-10-08 05:06:13"
+}
+
+/**
+	Copyright (c) 2019 Eran Rosenthal
+	
+	This program is free software: you can redistribute it and/or
+	modify it under the terms of the GNU Affero General Public License
+	as published by the Free Software Foundation, either version 3 of
+	the License, or (at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+	Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public
+	License along with this program. If not, see
+	<http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(_doc, _url) {
+	return 'newspaperArticle';
+}
+function doWeb(doc, url) {
+	var i;
+	var item = new Zotero.Item('newspaperArticle');
+	
+	item.title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	item.publicationTitle = 'Globes';
+	item.url = url;
+	item.abstractNote = ZU.xpathText(doc, '//meta[@name="description"]/@content');
+	item.section = ZU.xpathText(doc, '//meta[@property="article:section"]/@content');
+	var tags = ZU.xpath(doc, '//meta[@property="article:tag"]');
+	for (i = 0; i < tags.length; i++) {
+		item.tags.push(tags[i].content);
+	}
+
+	var json = ZU.xpathText(doc, '//script[@type="application/ld+json"]');
+	var data = JSON.parse(json);
+
+	if (data) {
+		if (data.author && Array.isArray(data.author)) {
+			for (i = 0; i < data.author.length; i++) {
+				// note the author sometimes include date in form of Month Year
+				// and possibly Day Month Year
+				var author = data.author[i].name;
+				var isDate = /19[0-9]{2}$/.test(author)
+					|| /2[0-9]{3}$/.test(author);
+				if (!isDate) {
+					author = Zotero.Utilities.cleanAuthor(author, 'author');
+					item.creators.push(author);
+				}
+			}
+		}
+		if (data.inLanguage) {
+			item.language = data.inLanguage;
+		}
+
+		if (data.datePublished) {
+			item.date = ZU.strToISO(data.datePublished);
+		}
+	}
+
+	item.attachments.push({
+		title: "Globes Snapshot",
+		document: doc,
+		mimeType: "text/html"
+	});
+	item.complete();
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://www.globes.co.il/news/article.aspx?did=1001302127",
+		"items": [
+			{
+				"itemType": "newspaperArticle",
+				"title": "הקרטל התאילנדי שובר שיאים של אכזריות. האם אדם אחד יצליח לעצור אותם?",
+				"creators": [
+					{
+						"firstName": "ג'ושוע",
+						"lastName": "האמר",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "",
+						"lastName": "GQ",
+						"creatorType": "author"
```

Comment:
Is this a real author or should we rather filter this one out?

Reply:
this a translated article (original aticle:  https://www.gq.com/story/hunting-rhino-horn-cartel-of-thailand ) and the credits are to GQ. 
I'm not sure how to determine it programmatically (other than heuristics of having both firstName and lastName?)

Reply:
I would suggest only to not take it as author if it is exactly `GQ`. (If we have more cases like this this, then we may see better about which pattern might be suitable.)

Reply:
Seems the site changes the metadata in that item - updating the test itself